src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql

```diff
@@ -0,0 +1,14 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:moto_masking_account_update_permission
```

Comment:
Changeset ID doesn’t quite match the permission: should be `uk.gov.pay:moto_mask_input_update_permission`.

Reply:
ok will make that change